data/json/npcs/TALK_COMMON_MISSION.json

```diff
@@ -223,10 +233,8 @@
     "responses": [
       {
         "text": "Glad to help.  I need no payment.",
-        "topic": "TALK_NONE",
         "effect": "clear_mission",
-        "mission_opinion": { "trust": 4, "value": 3 },
-        "opinion": { "fear": -1, "anger": -1 }
+        "success": { "topic": "TALK_NONE", "mission_opinion": { "trust": 4, "value": 3 }, "opinion": { "fear": -1, "anger": -1 } }
```

Comment:
There's no trial here, so the previous version should have been fine.

Reply:
I debugged the code when applying this, and without putting the opinion and mission_opinion into a success block, the JSON parsing does not pick them up at all. This change absolutely affected the outcome.

Reply:
Okay, good to know.  I'll fix that at some point.